grunt/config/less.js

```diff
@@ -4,12 +4,12 @@ module.exports = function (grunt, options) {
             options:{
                 baseUrl: '<%= sourcedir %>',
                 mandatory: [
-                    '<%= sourcedir %>core/less/*.less'
+                    '<%= sourcedir %>core/less/**/*.less'
                 ],
                 src: [
-                    '<%= sourcedir %>menu/<%= menu %>/**/*.less',
                     '<%= sourcedir %>components/**/*.less',
                     '<%= sourcedir %>extensions/**/*.less',
+                    '<%= sourcedir %>menu/<%= menu %>/**/*.less',                    
```

Comment:
My +1 still stands but we may want to discuss at some point whether extensions should appear _after_ the menu in the compile order.